mythx_cli/fuzz/run.py

```diff
@@ -18,6 +18,27 @@
 time_limit_seconds = 3000
 
 
+class IDE(Enum):
+    BROWNIE = 'brownie'
+    HARDHAT = 'hardhat'
+    TRUFFLE = 'truffle'
+    SOLIDITY = 'solidity'
+
+
+def determine_ide(build_directory) -> IDE:
+    root_dir = Path(build_directory).parent
```

Comment:
The root dir isn't necessarily the parent of the build dir (in the brownie case for example).

Maybe we should use pwd here?

Reply:
yeah, great suggestion. Will refactor